lib_xua/src/core/audiohub/xua_audiohub.xc

```diff
@@ -577,7 +576,9 @@ void testct_byref(chanend c, int &returnVal)
         returnVal = 1;
 }
 
-#if (XUA_DFU_EN == 1)
+#if (XUA_DFU_EN == 1) && ((NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0))
```

Comment:
Not quite sure why this needs to be guarded by ifdefs? Won't the linker remove it if not called?

Reply:
Yeah but you get a warning about not being used.